packages/apps/transfer/src/pages/transfer/account-transactions/index.tsx

```diff
@@ -63,42 +64,39 @@ const CheckTransactions: FC = () => {
     if (router.isReady) {
       getAndSetTransactions(
         router.query.network as Network,
-        router.query.chain as string,
+        chainID,
         router.query.account as string,
       ).catch((e) => {
         debug(e);
       });
     }
   }, [router.query.network, router.query.chain, router.query.account]);
```

Comment:
missing chainId, isready from dependencies of useEffect

Reply:
Wasn't really part of this PR, I can make a separate one where I get rid of all the warnings